modules/core/src/core-layers/text-layer/font-atlas.js

```diff
@@ -36,33 +36,39 @@ export function makeFontAtlas(
   // measure texts
   let row = 0;
   let x = 0;
+  let fontHeight = 0;
   const mapping = {};
 
   Array.from(characterSet).forEach(char => {
-    const {width} = ctx.measureText(char);
+    const {width, fontBoundingBoxDescent} = ctx.measureText(char);
+    if (!fontHeight) {
```

Comment:
Nit: (on comparison style) in JavaScript there is always some potential for confusion between zero and falsy values.

Since `fontHeight` is initialized as an integer `0` (rather than `undefined` or `null`), I would explicitly compare against `0` here, just to make it extra clear what the intention was.